nUpdate.WPFUserInterface/UpdaterUI.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright © Sascha Patschka 2018
+
+using System;
+using System.IO;
+using nUpdate.Internal.Core.Localization;
+using nUpdate.ServiceInterfaces;
+using nUpdate.WPFUserInterface.Services;
+
+// ReSharper disable once CheckNamespace
+namespace nUpdate.Updating
```

Comment:
Please, as already mentioned, make namespaces project dependent.

Reply:
In your ProvideTAP you have exactly the same Namespace in use - without the Projectname!!

Reply:
Yes, the reason is that ProvideTAP is a dependent part of the nUpdate library. But we should handle the WPFUserInterface as a separate project due to the architecture.

Reply:
Yes, we must change the architekture or divide the nUpdate.Shared Assembly...

Reply:
That is what I thought about. It's a bit of a problem, if we handle the WPF user interface separately.